syft/workers/base.py

```diff
@@ -251,7 +255,10 @@ def recv_msg(self, bin_message: bin) -> bin:
             self.msg_history.append(bin_message)
 
         # Step 0: deserialize message
-        (msg_type, contents) = sy.serde.deserialize(bin_message, worker=self)
+        msg = sy.serde.deserialize(bin_message, worker=self)
```

Comment:
Same as above for my question about tuples

Reply:
same answer as above